api/src/controllers/robiot.controller.ts

```diff
@@ -1,30 +1,44 @@
-import { Request, Response, NextFunction } from 'express';
+import { NextFunction, Request, Response } from 'express';
```

Comment:
Codacy found an issue: ['NextFunction' is defined but never used.](https://app.codacy.com/gh/Ca-Cybersecurity-Robiot/robiot/pullRequest?prid=5941216)